cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/datastore/DatabaseManagerTest.java

```diff
@@ -47,11 +44,11 @@ public void tearDown() {
         FileUtils.deleteQuietly(TEST_PATH);
     }
 
-    @Test(expected = IllegalArgumentException.class)
-    public void nonWritableDatastoreManagerPathThrows() throws DatastoreNotCreatedException {
+    @Test(expected = DatastoreNotOpenedException.class)
+    public void nonWritableDatastoreManagerPathThrows() throws DatastoreNotOpenedException, IOException {
         File f = new File(TEST_PATH, "c_root_test");
         try {
-            f.mkdir();
+            FileUtils.forceMkdir(f);
```

Comment:
Why are we forcing the creation of the directory? Surely it should be writeable?